documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1008,6 +1008,78 @@ The message value must be set to `null` to instruct Kafka to remove _all message
 To make this possible, by default, {prodname}'s Oracle connector always follows a _delete_ event with a special _tombstone_ event that has the same key but `null` value.
 You can change the default behavior by setting the connector property xref:oracle-property-tombstones-on-delete[`tombstones.on.delete`].
 
+// Type: continue
+[[postgresql-truncate-events]]
+=== _truncate_ events
+
+A  _truncate_ change event signals that a table has been truncated.
+The message key is `null` in this case, the message value looks like this:
+
+[source,json,indent=0,subs="+attributes"]
+----
+{
+    "schema": { ... },
+    "payload": {
+        "source": { // <1>
+            "version": "{debezium-version}",
+            "connector": "postgresql",
+            "name": "PostgreSQL_server",
+            "ts_ms": 1559033904863,
+            "snapshot": false,
+            "db": "postgres",
+            "schema": "public",
+            "table": "customers",
+            "txId": 556,
+            "lsn": 46523128,
+            "xmin": null
+        },
+        "op": "t", // <2>
+        "ts_ms": 1559033904961 // <3>
+    }
+}
+----
+
+.Descriptions of _truncate_ event value fields
+[cols="1,2,7",options="header"]
+|===
+|Item |Field name |Description
+
+|1
+|`source`
+a|Mandatory field that describes the source metadata for the event. In a _truncate_ event value, the `source` field structure is the same as for _create_, _update_, and _delete_ events for the same table, provides this metadata:
+
+* {prodname} version
+* Connector type and name
+* Database and table that contains the new row
+* Schema name
+* If the event was part of a snapshot (always `false` for _delete_ events)
+* ID of the transaction in which the operation was performed
+* Offset of the operation in the database log
+* Timestamp for when the change was made in the database
+
+|2
+|`op`
+a|Mandatory string that describes the type of operation. The `op` field value is `t`, signifying that this table was truncated.
+
+|3
+|`ts_ms`
+a|Optional field that displays the time at which the connector processed the event. The time is based on the system clock in the JVM running the Kafka Connect task.  +
++
+In the `source` object, `ts_ms` indicates the time that the change was made in the database. By comparing the value for `payload.source.ts_ms` with the value for `payload.ts_ms`, you can determine the lag between the source database update and {prodname}.
+
+|===
+
+In case a single `TRUNCATE` statement applies to multiple tables,
+one _truncate_ change event record for each truncated table will be emitted.
```

Comment:
Could you add a test for this case?

Reply:
Seems this is wrong. You cannot do that in ORACLE (in postgres you can). I'll remove this block.